app/admin/subsite.rb

```diff
@@ -0,0 +1,27 @@
+ActiveAdmin.register Subsite do
+  permit_params :name, :site_id
+
+  index do
+    column :name
+    column :site do |s|
+      "#{s.property&.name} - #{s.name}"
```

Comment:
Right.  That should be `s.site&.propery&.name` ... odd that it worked when I tried it. :-/